src/constants.ts

```diff
@@ -7,7 +7,9 @@ import fi from './assets/localization/languages/fi.json'
 import fr from './assets/localization/languages/fr.json'
 import hu from './assets/localization/languages/hu.json'
 import it from './assets/localization/languages/it.json'
+import nb from './assets/localization/languages/nb.json'
```

Comment:
Not sure if there are collisions, but we should probably adhere to the normal locale standard `no_nn /no_nb`, like done for `pt-br`

Reply:
Doesn't matter to me😊  Preferably just pick a standard and stick to it

Reply:
Although I agree that ideally we should use the same standard for everything, that seems to not be the case for Home Assistant (or they are using a standard that I don't know). I just checked changing the languages on Home Assistant and the values returned for both cases:

- Norsk Bokmål = `nb`
- Norsk Nynorsk = `nn`

So if we want language to work directly with Home Assistant we are going to have to follow what they use

Reply:
It seems to be a bit more complex and it's using multiple standards https://stackoverflow.com/a/21853099

Reply:
@bjornomy GH is not letting me to request for a re-review from you (not sure why), but feel free to add any feedback to the conversation or approve and I'll merge it

Reply:
As a norwegian my self, I've dealt with this unique problem (very few nations has more than 1 offical written language), and that's why I mentioned it.

After my rewrite in #64, any approach should work, as `Intl.DateTimeFormatter` does some magic here.
I tested using `nn` and `nb` as language, and it does work on my branch. As such we could probably use those.